ledger/eval/eval.go

```diff
@@ -1715,11 +1708,11 @@ transactionGroupLoop:
 		select {
 		case <-ctx.Done():
 			return ledgercore.StateDelta{}, ctx.Err()
-		case err, open := <-txvalidator.done:
+		case err1, open := <-txvalidator.done:
 			if !open {
 				break
 			}
-			if err != nil {
+			if err1 != nil {
 				return ledgercore.StateDelta{}, err
```

Comment:
```suggestion
				return ledgercore.StateDelta{}, err1
```

Reply:
+1

Reply:
Addresses in latest, this was what caused test failure too (so good that was picked up automatically!).